### ISSUE TITLE
Galaxy NG Docker Images

### ISSUE THREAD

**Issue by m-soltani** [pulp/pulp-oci-images#697]
Dear Team,

We have been relying on the Docker images provided by pulp to host our infrastructure and offer Ansible Galaxy as a central service to our teams.

Three weeks ago, a notification was added to the Pulp documentation website stating:

_"We are no longer building any newer image versions of galaxy-ng>=4.10. We will still continue to build new Z-updates for existing tags but going forward the stable and latest tags are now discontinued."_

This is a significant issue for us, as we cannot upgrade our infrastructure to the latest versions (currently 4.10.0 and 4.10.1) since Galaxy Docker images with these tags are no longer being published as well as `latest` and `nightly` which are also discontinued as per announcement.

Is the expectation that anyone interested should start building docker images on their own?

Could you please provide some insights into this decision and explain why you are discontinuing the publication of new Docker image versions for these particular tags?

**Comment by mdellweg** [comment]
> Is the expectation that anyone interested should start building docker images on their own?

TL;DR: Yes.

This team does not own the galaxy-ng codebase. After all, we realized that we neither have the capacity, nor the expertise to produce these containers with any level of confidence. Continuing to do so would be a disservice to the community.
We added a section to the docs that outline how to customize the image building process:
https://pulpproject.org/pulp-oci-images/docs/admin/guides/build-your-own-pulp-image/
I hope this helps you and your team.

**Comment by m-soltani** [comment]
Thanks for the link. For someone who is not a core developer of this product, it is equally challenging to work on docker images. You basically need to understand both pulp and galaxy-ng in-depth to be able to provide a docker image with the expected confidence level. I won't complain though, after all, this is an open-source project and appreciate your efforts here.

**Comment by mdellweg** [comment]
Thank you for the kind words. I know our documentation is mainly written by the same developers and may be hard to read at times. (We love to get constructive feedback there too. [hint])
